src/kernel/capinv.c

```diff
@@ -153,13 +156,14 @@ kmem_deact_pre(struct cap_header *ch, struct captbl *ct, capid_t pgtbl_cap,
 	if (!CAP_TYPECHK(cap_pt, CAP_PGTBL)) cos_throw(err, -EINVAL);
 
 	/* get the pte to the cos frame. */
-	*p_pte = pgtbl_lkup_pte(cap_pt->pgtbl, cosframe_addr, &flags);
+	*p_pte =pgtbl_lkup_pte(&(cap_pt->pgtbl), cosframe_addr, &size_order, &flags);
```

Comment:
This page table implementation will later be moved into the Cortex-M specific chal.